src/game.cpp

```diff
@@ -9387,18 +9415,22 @@ void game::butcher()
             }
             break;
         case BUTCHER_CORPSE: {
+            auto cut_time = [&]( const char *text, enum butcher_type bt ) {
+                return string_format( text, to_string_clipped( time_duration::from_turns( butcher_time_to_cut( u,
+                                      items[corpses[ret]], bt ) / 100 ) ) );
+            };
             uilist smenu;
             smenu.desc_enabled = true;
             smenu.text = _( "Choose type of butchery:" );
-            smenu.addentry_desc( BUTCHER, true, 'B', _( "Quick butchery" ),
+            smenu.addentry_desc( BUTCHER, true, 'B', cut_time( _( "Quick butchery (%s)" ), BUTCHER ),
                                  _( "This techinque is used when you are in a hurry, but still want to harvest something from the corpse.  Yields are lower as you don't try to be precise, but it's useful if you don't want to set up a workshop.  Prevents zombies from raising." ) );
```

Comment:
Not yours but while you're here can you fix the spelling error?

"techinque" vs "technique"